app/client/components/wizardRouterAdapter.tsx

```diff
@@ -15,7 +15,7 @@ interface RouteableProps extends RouteComponentProps {
 }
 
 interface RoutablePropsWithChildren extends RouteableProps {
-  children?: any; // TODO ReactElement<RouteableProps> | ReactElement<MultiRouteableProps>[];
+  children?: any; // JSX.Element | React.Component<RouteableProps> | React.FC<RouteableProps> | React.Component<MultiRouteableProps> | React.FC<MultiRouteableProps>[];
```

Comment:
Can you add some detail as to why you're changed this comment? Did you meant to remove the `TODO` keyword?